### PR TITLE
Change instances of history to activity

### PR DESCRIPTION
**Pull request by theresaslee**
Addresses the frontend part of #427.

TODO:
 - [ ] change history api to activity. Did not do this yet since that doesn't exist.

### CODE REVIEW

File: client/app/components/group/_activity/_activityDetail/activityDetail.spec.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import StoreHistoryModule from "./storeHistory";
+import ActivityDetailModule from "./activityDetail";
```
Comment (djahnie) [inline client/app/components/group/_activity/_activityDetail/activityDetail.spec.js]: Why did you decide to change `storeHistory` to `activityDetail` and not `storeActivity`? I would have found the second one more intuitive. Did I miss something?

Comment (theresaslee) [reply client/app/components/group/_activity/_activityDetail/activityDetail.spec.js]: 🙈 I have no idea why I did that. Sorry about that!

Comment (theresaslee) [reply client/app/components/group/_activity/_activityDetail/activityDetail.spec.js]: Actually, looking at this more closely, I think Github might have mistaken where the changes were coming from when I renamed the files.
I believe [this](https://github.com/yunity/foodsaving-frontend/blob/master/client/app/components/group/_history/_historyDetail/historyDetail.spec.js) is the actual file I converted.

Comment (nicksellen) [reply client/app/components/group/_activity/_activityDetail/activityDetail.spec.js]: How confusing! I did a diff of the file trees [1] and all looks good there, and all the file contents look good too. Silly computers :)

[1] https://gist.github.com/nicksellen/1bef6d29edeaf2cb6aabbcffbe86efa0

Comment (djahnie) [reply client/app/components/group/_activity/_activityDetail/activityDetail.spec.js]: Weird... good to know that stuff like this can happen... 😅

**Comment by tiltec** [comment]
Looks good, though I didn't look in detail! 

We need to take care again with the translation files - they are managed on transifex. If we just update them in the repo, they will get overwritten on the next translation update.

The only suitable process to change translation keys seems to me:

1. pull up-to-date translation files from transifex
2. rename the key in all translation files
3. push the changes back to transifex
4. hope that no translator is active between (1) and (3), otherwise their changes got lost
5. make a commit and push to github

Also: the backend part seems harder to me and there is currently no one working on it. How should we proceed?

We can merge this PR without the API change (as you prepared it), but if no one is following up on the backend side, we will get inconsistent naming. That's why, I'd merge only when the backend is ready, too.

**Comment by tiltec** [comment]
Closing this because we merged the vuejs/quasar-based rewrite to master, which has completely different structure.

**Pull request closed without merging**